stories/SelectField.stories.js

```diff
@@ -9,80 +9,119 @@ const options = [
     { value: '2', label: 'Beware the power of option 2' },
 ]
 
-const createSelectField = override => (
-    <div style={{ width: '200px' }}>
-        <SelectField
-            name="select-field"
-            label="Label text"
-            onChange={noop}
-            list={[]}
-            {...override}
-        />
-    </div>
+class Wrapper extends Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            value: this.props.value || '',
+        }
+    }
+
+    onChange = (name, value) => {
+        this.setState({ value })
+    }
+
+    render() {
+        const { value, ...props } = this.props
+
+        return (
+            <div style={{ width: '200px' }}>
```

Comment:
I think we can remove this wrapper `div` so that the `SelectField` and `InputField` get the same amount of space in the demo.